eng/targets/Helix.targets

```diff
@@ -8,6 +8,21 @@
     <HelixPreCommand Include="call RunPowershell.cmd mssql\InstallSqlServerLocalDB.ps1 || exit /b 1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TestDependsOnPlaywright)' == 'true' AND '$(IsWindowsHelixQueue)' == 'true'">
+    <SkipHelixQueues>Windows.7.Amd64.Open</SkipHelixQueues>
```

Comment:
Win7 doesn't support Enable-WindowsOptionalFeature / Get-WindowsOptionalFeature needed to install ServerMediaFeature req of playwright

Reply:
Win81 hangs sometimes still (probably requires a reboot), can reenable once images are updated to have the feature installed by default